stdlib/public/Reflection/MetadataSource.cpp

```diff
@@ -96,11 +97,9 @@ class PrintMetadataSource
   }
 };
 
-void MetadataSource::dump() const {
-  dump(stderr, 0);
-}
+void MetadataSource::dump() const { dump(std::cout, 0); }
```

Comment:
This, and various others, should be `cerr` to match the previous code, although I'm not sure offhand why we use one versus the other.